server/helpers/utils/utils.go

```diff
@@ -419,10 +421,81 @@ func FormatToTitleCase(s string) string {
 	return c.String(s)
 }
 func ExtractFile(filePath string, destDir string) error {
+
 	if utils.IsTarGz(filePath) {
 		return utils.ExtractTarGz(destDir, filePath)
 	} else if utils.IsZip(filePath) {
 		return utils.ExtractZip(destDir, filePath)
 	}
 	return utils.ErrExtractType
 }
+func ConvertToJSONCompatible(data interface{}) interface{} {
```

Comment:
@Jougan-0 **In this same file look a few lines above, you will find a function which is meant to do the same thing.**

Before adding logic, check if it already exists.